_includes/tutorials/time-concepts/ksql/markup/dev/make-application.adoc

```diff
@@ -0,0 +1,15 @@
+Achieving event-time semantics typically requires embedding timestamps into the data record at the time it is produced.
```

Comment:
@awalther28 the reasoning was that `event` is a touch more abstract/conceptual, whereas `record` refers more to the actual data/fields/metadata/etc...and because this KT is talking about the actual data and fields within the payload, there are places where `record` seemed to fit better than `event`.  However, in writing his book, @bbejeck has probably stronger leanings on when to use which term -- any thoughts @bbejeck ?

Reply:
Gotcha, with those definitions I think `record` makes sense here. I am curious to hear @bbejeck 's opinion.

Reply:
Yeah, in this case, I think using the term `data record`,  or just `record` makes sense. I would do the same.  IMO the term `event` is more about something that happens and the details captured in a physical `record`.  

While the terms `event record`, `data record,` or `record` are somewhat synonymous,  my personal preference is to talk about events as the "thing that happened" and then refer to the results as a `record`, as it's implied they are `event records`.  To me, the main point is to emphasize that Kafka captures the event when it occurs (in most cases) then makes it available almost immediately.  I may have gone off-topic here a little, but that's my  2 cents.

Reply:
@bbejeck 👍 thanks for the insight.  Makes sense